lib/cluster/cluster.c

```diff
@@ -103,12 +103,13 @@ pcmk_cluster_connect(crm_cluster_t *cluster)
  * \param[in,out] cluster  Cluster object to disconnect
  */
 void
```

Comment:
We could return a standard return code here (`pcmk_rc_ok` usually, or something like `EPROTONOSUPPORT` for an unsupported cluster type). The main advantage of that is that it's extensible in the future.

Currently, the return code wouldn't matter: either we disconnected successfully, or there was no active connection (either because the `crm_cluster_t` object was already disconnected or because the cluster type is unsupported and we never could have connected).

Reply:
good idea